Kickstarter-iOS/AppDelegate.swift

```diff
@@ -269,10 +274,27 @@ final class AppDelegate: UIResponder, UIApplicationDelegate {
   fileprivate func goToMessageThread(_ messageThread: MessageThread) {
     self.rootTabBarController?.switchToMessageThread(messageThread)
   }
+
+  private func findRedirectUrl(_ url: URL) {
+    let session = URLSession(configuration: .default, delegate: self, delegateQueue: nil)
+    let task = session.dataTask(with: url)
+    task.resume()
+  }
 }
 
 extension AppDelegate : BITHockeyManagerDelegate {
   func crashManagerDidFinishSendingCrashReport(_ crashManager: BITCrashManager!) {
     self.viewModel.inputs.crashManagerDidFinishSendingCrashReport()
   }
 }
+
+extension AppDelegate: URLSessionTaskDelegate {
+  public func urlSession(_ session: URLSession,
+                         task: URLSessionTask,
+                         willPerformHTTPRedirection response: HTTPURLResponse,
+                         newRequest request: URLRequest,
+                         completionHandler: @escaping (URLRequest?) -> Void) {
+    request.url.doIfSome(self.viewModel.inputs.foundRedirectUrl)
+    completionHandler(nil)
```

Comment:
this stops the url session from following the redirect so to not download any extra data